lang/es/filter_oembed.php

```diff
@@ -15,16 +15,16 @@
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
 
 /**
- * Language strings for component 'filter_ilos_oembed'
+ * Language strings for component 'filter_vidgrid_oembed'
  *
- * @package   filter_ilos_oembed
- * @copyright 2012 Matthew Cannings; modified 2015 by Microsoft Inc.; modified 2016 by Ilos Co
+ * @package   filter_vidgrid_oembed
+ * @copyright 2012 Matthew Cannings; modified 2015 by Microsoft Inc.; modified 2018 by VidGrid
```

Comment:
don't think it's a huge deal but if this copyright is displayed anywhere, our official name is "VidGrid Inc."